src/olympia/devhub/templates/devhub/addons/submit/upload.html

```diff
@@ -38,7 +56,7 @@
         {{ new_addon_form.errors.supported_platforms }}
         {{ new_addon_form.supported_platforms }}
       </div>
-      {{ new_addon_form.upload }}
```

Comment:
I looked at git log and tried various scenarios locally and it doesn't seem to be needed at all, my guess is that it was just not removed when the upload flow last changed... 5 years ago.